### PR TITLE
Prep feat io

### PR DESCRIPTION
**Pull request by nidajanu**

### CODE REVIEW

File: cytoxnet/dataprep/featurize.py [hunk 1]
```diff
@@ -1,2 +1,101 @@
...
+    else:
+        print('Column ID does not contain inchi or smiles')
+
+    # add the mol objects to the dataframe
+    dataframe['Mol'] = mol_obj_list
+
+    return dataframe
+
+
+def featurize(dataframe, MolcolumnID='Mol', method='CircularFingerprint'):
```
Comment (EvanKomp) [inline cytoxnet/dataprep/featurize.py:47]: Depechem might have most of this done, have a look at FeaturizerTransformer

File: cytoxnet/dataprep/featurize.py [hunk 2]
```diff
@@ -1,2 +1,101 @@
...
+
+    # Check that set contains Mol objects
+    assert isinstance(object_in_set, rdkit.Chem.rdchem.Mol)
+
+    featurizer =  getattr(deepchem.feat, method)
+
+    # molecules format optins-  rdkit.Chem.rdchem.Mol / SMILES string / iterable
+    # either a loop or pass an iterable set of Mol
+    # iterable = convert column of Mol to array
+    features = featurizer.featurize(iterable)
```
Comment (EvanKomp) [inline cytoxnet/dataprep/featurize.py]: Series (pandas column) should work I think

File: cytoxnet/dataprep/featurize.py [hunk 3]
```diff
@@ -1,2 +1,101 @@
...
+    if 'inchi' in strcolumnID.lower():
+        # convert InChI to Mol
+        for inchi in column:
+            mol_obj = rdkit.Chem.inchi.MolFromInchi(inchi)
+            #append to array first here probably
+    elif 'smiles' in strcpolumnID.lower():
+        # convert SMILES to Mol
+        for smiles in column:
+            mol_obj = MolFromSmiles(smiles)
+            # probably append to an array
```
Comment (EvanKomp) [inline cytoxnet/dataprep/featurize.py]: Might be worth at some point having the type (smiles or inchi) be specifiable by the user, parsing the string will fail if for some reason the name is not in the column name

**Review by EvanKomp — APPROVED** [review]
Looks good!

**Review by roryjm — APPROVED** [review]

**Pull request merged**